S/Spasm/bundled/patches/zap-gmp-test.patch

```diff
@@ -0,0 +1,30 @@
+diff --git givaro.orig/macros/gmp-check.m4 givaro/macros/gmp-check.m4
+--- givaro.orig/macros/gmp-check.m4
++++ givaro/macros/gmp-check.m4
+@@ -105,21 +105,11 @@ AC_DEFUN([GIV_CHECK_GMP], [
+ 		exit 1
+ 	])
+ 
+-	AC_MSG_CHECKING([whether gmp version is at least $min_gmp_release])
+-	AC_TRY_RUN(
+-		[ 
+-			#include <cstddef>
+-			#include <gmp.h>
+-			int main () {
+-				return (__GNU_MP_RELEASE < $min_gmp_release);
+-			}
+-		],
+-		[ AC_MSG_RESULT(yes)
+-		],
+-		[ AC_MSG_RESULT(no)
+-		  AC_MSG_ERROR(your GMP is too old. GMP release >= $min_gmp_release needed)
+-		  exit 1]
+-	)
++	### Major hack! we need to add -lrt sometimes, let's do it here
```

Comment:
Why not setting `LIBS` in the build script instead?

Reply:
Agree -- that would be better. I'll do it on the givaro branch, where this patch is now applied.